README.md

```diff
@@ -1,103 +1,68 @@
- - The datastructure is a global mesh of append-only logs, which can support new types of data (not just "mail").
- - Users are not bound to one server/host (what we call "Pubs") and do not have to trust the servers.
- - It's very easy to setup and maintain your own Pub.
+1. Choose a pub from the [pub list][pub-list] and copy an invite code.
+2. Open Patchwork and select *Join Pub*.
+3. Paste the invite code and select *Redeem Invite*.
 
-It's better than Twitter and Facebook because:
+You're done! Check out `#new-people` to see who else has recently joined.
 
- - Private messages are end-to-end encrypted, always.
- - The software runs on your device, so there's nobody tracking your browsing.
- - The application code is FOSS, so you're free to fork or write new applications without a gatekeeper setting terms.
- - Data is saved to your disk, and so the application works offline.
- - You can sync directly with friends over Wi-Fi.
+## Installation
 
-## Pubs
+Get the [latest release from GitHub][latest] or install with a package manager.
 
-In order to gossip outside your local network, you'll need to connect to a [Pub](https://www.scuttlebutt.nz/concepts/pub.html).
-
-You can find an invite to an available pub at the [Pub servers registry](https://github.com/ssbc/ssb-server/wiki/Pub-Servers).
-
-Get an invite, then in `patchwork`: click "+ Join Pub" and paste the invite code.
-
-This will cause you to follow the Pub and vice versa. If you haven't synchronized to this social network yet, prepare to wait a few minutes while your local server synchronizes.
-
-## Install
-
-### Download installers for Windows, macOS and Linux
-
-See [patchwork/releases](https://github.com/ssbc/patchwork/releases)
-
-### Or build from source
-
-With node (>= 4) and npm (>= 3) installed:
-
-```shell
-$ git clone https://github.com/ssbc/patchwork
-$ cd patchwork
-$ npm install
-$ npm start
-```
-
-You'll need some more dependencies to build.
-
-On Debian Linux:
+With [npm][npm]:
 
 ```shell
-sudo apt-get install g++ m4 libtool automake libgconf-2-4   # build tools
-sudo apt-get install libxext-dev libxtst-dev libxkbfile-dev # spellchecker headers
+npm install --global ssb-patchwork
 ```
 
-On Fedora Linux:
+With [yarn][yarn]:
 
 ```shell
-sudo dnf install libXext-devel libXtst-devel libxkbfile-devel gcc-c++ m4 automake libtool
+yarn global add ssb-patchwork
 ```
 
-On macOS:
+With [yay][yay]:
 
 ```shell
-brew install libtool automake # build tools
+yay -S ssb-patchwork
 ```
 
-## Code of conduct
+Need something else? Check out [`INSTALL.md`][install] for more information.
 
-Please note that this project is released with a [Contributor Code of Conduct](code-of-conduct.md). By participating in this project you agree to abide by its terms.
+## See Also
 
-## Contributing
+- [patchbay][patchbay]
+- [ssb-server][ssb-server]
+- [manyverse][manyverse]
 
-Please read the [Patchwork Contribution Guidelines](contributing.md) before opening a pull request.
```

Comment:
are we sure we want to remove the section for contributors?

Reply:
Oops, will fix!